src/main/java/twilightforest/client/renderer/blocks/RenderBlockTFFieryMetal.java

```diff
@@ -0,0 +1,287 @@
+            } else icon = block.fieryCore[2];
+        } else {
+            if (neighboursNS) icon = block.fieryCore[1];
+            else icon = block.fieryCore[0];
+        }
+        renderblocks.renderFaceZPos(block, x, y, z, icon);
+        renderblocks.renderFaceZNeg(block, x, y, z, icon);
+
+        // Y faces
+        if (neighboursEW) {
+            if (neighboursNS) {
+                if (neighbourNE || neighbourNW || neighbourSE || neighbourSW) icon = block.fieryCore[4];
+                else icon = block.fieryCore[3];
+            } else icon = block.fieryCore[2];
+        } else {
+            if (neighboursNS) icon = block.fieryCore[1];
+            else icon = block.fieryCore[0];
+        }
+        renderblocks.renderFaceYPos(block, x, y, z, icon);
+        renderblocks.renderFaceYNeg(block, x, y, z, icon);
+    }
+
+    public static void renderInvJar(RenderBlocks renderblocks, Block par1Block, int meta) {
+        Tessellator tessellator = Tessellator.instance;
+
+        GL11.glTranslatef(-0.5F, -0.5F, -0.5F);
+        GL11.glColor4f(1.0F, 1.0F, 1.0F, 1.0F);
+
+        float p = 1F / 16F;
+
+        // middle
+        renderblocks.setRenderBounds(p, p, p, 1 - p, 1 - p, 1 - p);
+        renderInvBlock(renderblocks, par1Block, meta, tessellator);
+
+        // outline
+        renderblocks.setRenderBounds(0, 0, 0, 1, 1, 1);
+        renderInvOutline(renderblocks, (BlockTFCompressed) par1Block, meta, tessellator);
+
+        GL11.glTranslatef(0.5F, 0.5F, 0.5F);
+
+        par1Block.setBlockBoundsForItemRender();
+    }
+
+    public static void renderInvBlock(RenderBlocks renderblocks, Block par1Block, int meta, Tessellator tessellator) {
+        tessellator.startDrawingQuads();
+        tessellator.setNormal(0.0F, 1.0F, 0.0F);
+        renderblocks.renderFaceYNeg(par1Block, 0.0D, 0.0D, 0.0D, par1Block.getIcon(0, meta));
+        tessellator.draw();
+        tessellator.startDrawingQuads();
+        tessellator.setNormal(0.0F, 1.0F, 0.0F);
+        renderblocks.renderFaceYPos(par1Block, 0.0D, 0.0D, 0.0D, par1Block.getIcon(1, meta));
+        tessellator.draw();
+        tessellator.startDrawingQuads();
+        tessellator.setNormal(0.0F, 1.0F, 0.0F);
+        renderblocks.renderFaceXPos(par1Block, 0.0D, 0.0D, 0.0D, par1Block.getIcon(2, meta));
+        tessellator.draw();
+        tessellator.startDrawingQuads();
+        tessellator.setNormal(0.0F, 1.0F, 0.0F);
+        renderblocks.renderFaceXNeg(par1Block, 0.0D, 0.0D, 0.0D, par1Block.getIcon(3, meta));
+        tessellator.draw();
+        tessellator.startDrawingQuads();
+        tessellator.setNormal(0.0F, 1.0F, 0.0F);
+        renderblocks.renderFaceZNeg(par1Block, 0.0D, 0.0D, 0.0D, par1Block.getIcon(4, meta));
+        tessellator.draw();
+        tessellator.startDrawingQuads();
+        tessellator.setNormal(0.0F, 1.0F, 0.0F);
+        renderblocks.renderFaceZPos(par1Block, 0.0D, 0.0D, 0.0D, par1Block.getIcon(5, meta));
+        tessellator.draw();
```

Comment:
You can get away with a single draw call here, also the fiery metal block is almost completely dark when smooth lightning is turned off.

Reply:
> You can get away with a single draw call here
Are you sure? If I remember correctly, this function was taken unchanged from `RenderBlockTFPedestal`

> the fiery metal block is almost completely dark when smooth lightning is turned off
Is it dark when placed in world or in inventory?

Reply:
I'm 100% sure. 

It's fine in inventory but dark in world.

Reply:
I reduced draw calls to one, and, welp, it doesn't work, just as I thought

![image](https://github.com/GTNewHorizons/twilightforest/assets/58332807/451407cb-4087-43a6-835d-4fdcbafd707b)

![image](https://github.com/GTNewHorizons/twilightforest/assets/58332807/a52920f7-2258-42c8-959d-2508c38db18b)

Reply:
Did you remove all but the first `startDrawingQuads`?

Reply:
Oh, those should be removed as well, of course
Give me a second

Reply:
Ok, it works
Will fix it for other renderers as well and then upload

Reply:
The rest of the setNormal calls can be removed as well iirc